library/src/interactor_impl.cxx

```diff
@@ -833,6 +786,32 @@ bool interactor_impl::triggerCommand(std::string_view command)
   return false;
 }
 
+//----------------------------------------------------------------------------
+interactor& interactor_impl::addInteractionCommands(
+  std::string interaction, ModifierKeys modifiers, const std::vector<std::string>& commands)
+{
+  this->Internals->InteractionCommands[{ std::move(interaction), std::move(modifiers) }] = commands;
+  return *this;
+}
```

Comment:
No reason to move a unsigned char imo, but avoid a copy of std::vector is definitely needed.

```suggestion
interactor& interactor_impl::addInteractionCommands(
  std::string interaction, ModifierKeys modifiers, std::vector<std::string> commands)
{
  this->Internals->InteractionCommands[{ std::move(interaction), modifiers }] = std::move(commands);
  return *this;
}
```